gauges/logical_replication.go

```diff
@@ -0,0 +1,95 @@
+package gauges
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/ContaAzul/postgresql_exporter/postgres"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type slots struct {
+	SlotName     string  `db:"slot_name"`
+	IsSlotActive float64 `db:"active"`
+	SlotTotalLag float64 `db:"total_lag"`
+}
+
+// ReplicationSlotStatus returns the state of the replication slots
+func (g *Gauges) ReplicationSlotStatus() *prometheus.GaugeVec {
+	var gauge = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name:        "postgresql_replication_slot_status",
+			Help:        "State of the replication slots",
```

Comment:
```suggestion
			Help:        "Returns 1 if the slot is currently actively being used",
```
sounds better as an help message